### DISCUSSION TITLE
How to install MongoDB on RaspberryOS?

### DISCUSSION THREAD

**Discussion by raydacosta** [Pi4J/pi4j#346] [General]
How should I install MongoDB on RaspberryOS?
I'm trying but it says that the Raspberry architecture is not compatible

**Comment by taartspi** [comment]
https://www.mongodb.com/developer/products/mongodb/mongodb-on-raspberry-pi/

Did you use the information on the mongodb web ?   If you are not successful with these I would suggest you use their website to ask questions specific to their instructions.

**Reply by raydacosta** [reply]
Hi taartspi, thanks for getting back to us. I followed exactly what is in your link. But what happens, there they use Ubuntu server 20. For Ubuntu there is MongoDB. It worked, but then I ran into another problem. For Ubuntu, there is no PI4J V2. I solve one problem, but I fix another.
